.github/workflows/create-release.yml

```diff
@@ -40,10 +40,8 @@ jobs:
       run: |
         source python/etc/profile.d/conda.sh
         conda activate venv_sct
-        pip freeze | grep -v "-e git+" | grep -v "torch" > requirements-freeze.txt
+        pip freeze | grep -v "-e git+" > requirements-freeze.txt
         conda deactivate
-        echo "# Platform-specific torch requirements (See SCT Issue #2745)" >> requirements-freeze.txt
-        grep "torch" requirements.txt >> requirements-freeze.txt
```

Comment:
Hrmm... The reason I did the `torch` grepping initially is because our old `requirements.txt` file had some platform-specific branching paths, depending on whether the user is on macOS or Linux:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/129e537b83e445fe96aed8b1971d748d8424c25e/requirements.txt#L32-L35

However, if we just `pip freeze`, then we won't get those platform-specific statements:

```console
(venv_sct) joshua@tadpole:~/repos/spinalcordtoolbox$ pip freeze
absl-py==1.0.0
astor==0.8.1
[...]
torch==1.5.0+cpu
torchvision==0.6.0+cpu
[...]
```

So we would need a way to fetch these platform-specific torch requirements from the `setup.py`. :thinking:

Reply:
I wrongly assumed they weren't needed after we froze packages oops

Reply:
Fixed